docs/package.json

```diff
@@ -114,7 +114,7 @@
     "rimraf": "^3.0.0",
     "styled-components": "^5.0.0",
     "stylis": "^4.0.3",
-    "stylis-plugin-rtl": "^1.0.0",
+    "stylis-plugin-rtl": "^2.0.0",
```

Comment:
Are we using this with `styled-components` or just emotion?

Reply:
+1 for making the change in a different pull-request. The badge pull request was meant to be an example the other migrations pull requests can be based on. Turns out, it was great to find "holes" in the current approach 😁.

Reply:
We are using it for both, however at this moment we have a mismatch on the versions compatible with styled-components and emotion. Previously we were using the v1 with both. We are still using it with both, but styled-components v5 is compatible with v1... As emotion is our styled engine I am using the version that works for emotion at this moment. 

https://github.com/styled-components/stylis-plugin-rtl

![image](https://user-images.githubusercontent.com/4512430/100763657-c16bbe80-33f5-11eb-8f50-78e76232f7c4.png)

Reply:
Will be fixed as part of https://github.com/mui-org/material-ui/pull/23796 I merged the current branch here to test it